nipype/interfaces/afni/utils.py

```diff
@@ -642,6 +642,105 @@ def _format_arg(self, name, spec, value):
             return spec.argstr%(' '.join([i[0]+' -'+i[1] for i in value]))
         return super(CatMatvec, self)._format_arg(name, spec, value)
 
+
+class CenterMassInputSpec(CommandLineInputSpec):
+    in_file = File(
+        desc='input file to 3dCM',
+        argstr='%s',
+        position=-2,
+        mandatory=True,
+        exists=True,
+        copyfile=True)
+    cm_file = File(
```

Comment:
this should have `hash_files=False` so that this value doesn't get hashed as a file.

Reply:
Ok !